tensorflow/tools/ci_build/ci_test_on_install.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# Get the command line arguments.
+CONTAINER_TYPE=$( echo "$1" | tr '[:upper:]' '[:lower:]' )
+
+# Validate command line arguments.
+if [ "$#" -gt 1 ] || [[ ! "${CONTAINER_TYPE}" =~ ^(cpu|gpu|android)$ ]]; then
+  >&2 echo "Usage: $(basename $0) CONTAINER_TYPE"
+  >&2 echo "       CONTAINER_TYPE can be 'CPU' or 'GPU'"
+  >&2 echo ""
+  >&2 echo "Example (run test-on-install on CPU):"
+  >&2 echo "$0 CPU"
+  exit 1
+fi
+
+
+COMMAND="/builds/test_on_install.sh ${CONTAINER_TYPE}"
```

Comment:
This certainly does not feel right. Copying 100 lines script to hardcode one line? Why simply not call the original script with this argument?